src/tbb/global_control.cpp

```diff
@@ -17,6 +17,7 @@
 #include "oneapi/tbb/detail/_config.h"
```

Comment:
Please update the last copyright year to 2024 for both files to satisfy the copyright check.